lib/Runtime/Library/CustomExternalWrapperObject.cpp

```diff
@@ -120,32 +140,25 @@ BOOL CustomExternalWrapperObject::EnsureInitialized(ScriptContext* requestContex
     return TRUE;
 }
 
-CustomExternalWrapperObject * CustomExternalWrapperObject::Clone(CustomExternalWrapperObject * source, ScriptContext * scriptContext)
+CustomExternalWrapperObject*
+CustomExternalWrapperObject::Copy(bool deepCopy)
```

Comment:
newline needed?

Reply:
no, in backend we do this a lot it's a habit... since rest of this doesn't do it i removed new line